Specs/DateTime/Dutch/DateTimeModel.json

```diff
@@ -727,6 +720,11 @@
               "timex": "2016-11-08T08:00",
               "type": "datetime",
               "value": "2016-11-08 08:00:00"
+            },
+            {
+              "timex": "2016-11-08T20:00",
```

Comment:
Here it's different, as "morgen" is specifying either morning and then there should be no 20h resolution; or tomorrow, then the date is wrong.
What is the corresponding output in English and German?

Reply:
"morgen" here should be interpreted as "tomorrow" (the date seems correct because the reference is 2016-11-07). The English counterpart is "I'll go back tomorrow 8:00am" which is resolved to "2016-11-08T08:00"